mne/io/meas_info.py

```diff
@@ -156,6 +158,7 @@ def _unique_channel_names(ch_names):
 
 
 # XXX Eventually this should be de-duplicated with the MNE-MATLAB stuff...
+# XXX: Digitization, Docstrings need a pass changing lists for Digitization
```

Comment:
Is this still true?

Reply:
The one I added? yes. I did not do the pass.